Revise filesystems page

willfurnass opened the pull request:
* Add info re different ways of handing chmod system calls on /shared
  filesystems
* Add more tables and cross-references
* Clarifications
* Formatting

willfurnass commented:
@unixnation: any comments?

willfurnass commented:
What would a 'backups' table look like for `/shared`?

anthonybrookfield commented on hpc/filestore.rst:
```diff
@@ -33,68 +20,114 @@ The storage areas differ in terms of:
++---------------------------+--------------------+---------------------------------------+
+| Frequency of snapshotting | Snapshots retained | Mirrored onto separate storage system |
++===========================+====================+=======================================+
+| Every 4 hours             | 10 most recent     | No                                    |
```
for /home snapshots - both the dailies and the 4 hourlies get sent to the backup storage system

anthonybrookfield commented on hpc/filestore.rst:
```diff
@@ -33,68 +20,114 @@ The storage areas differ in terms of:
++----------+---------------------+--------+----------------+---------------------+--------------------------------------------------------+---------------------------+
+| ShARC    | ``/fastdata``       | Lustre | None           | 669 TB              | ShARC's fastdata filesystem is accessible from Iceberg | 100Gb/s (*Omni-Path*)     |
++----------+---------------------+--------+----------------+---------------------+--------------------------------------------------------+---------------------------+
+| Iceberg  | ``/fastdata-sharc`` | Lustre | None           | 669 TB              | ShARC's fastdata filesystem is accessible from Iceberg | 10Gb/s Ethernet           |
```
For most Iceberg worker nodes network bandwidth to lustre is only 1Gb/s

anthonybrookfield commented on hpc/filestore.rst:
```diff
@@ -191,37 +235,49 @@ some applications can cause problems when accessing files/directories on ``/shar
+**Changing how attempts to change permissions are handled**: each ``/shared`` area can be configured so that
+
+1. Attempts to change file/directory mode bits fail (e.g. ``chmod +x /shared/mygroup1/myprogram.sh`` fails) (**default configuration per area**) **or**
+1. Attempts to change file/directory mode bits appear to succeed (e.g. ``chmod +x /shared/mygroup1/myprogram.sh`` does not fail but also does not actually change any permissions on the underlying file server) (**alternative configuration per area**)
```
this doesn't look to have formatted properly as separate items?

willfurnass commented:
Made requested changes and:
 - Added table on backup schedule for /shared areas - info correct?
 - Revised section on checking quotas - suggest folks just use `df -h $STORAGE_AREA` as it works for arbitrary /shared areas too.
 - Revised section on accessing snapshots.

willfurnass commented:
More changes following review comments:
 - fixed typos spotted by @desmfryan
 - defined snapshotting and mirrored backups in glossary
 - revised use of term 'backups' in filesystem page and added refs to glossary definitions
 - noted that *owners* of /shared areas need to request changes to how permissions handled
